docs/src/pipe.rst

```diff
@@ -68,15 +68,16 @@ API
 
     Bind the pipe to a file path (Unix) or a name (Windows).
 
-    ``flags`` must be zero. Returns ``UV_EINVAL`` for unsupported flags
-    without performing the bind operation.
+    ``flags`` must be zero or ``UV_PIPE_NO_TRUNCATE``. Returns ``UV_EINVAL``
+    for unsupported flags without performing the bind operation.
 
     Supports Linux abstract namespace sockets. ``namelen`` must include
     the leading nul byte but not the trailing nul byte.
 
     .. note::
         Paths on Unix get truncated to ``sizeof(sockaddr_un.sun_path)`` bytes,
-        typically between 92 and 108 bytes.
+        typically between 92 and 108 bytes, unless the ``UV_PIPE_NO_TRUNCATE``
+        flag is specified, in which case an ``UV_EINVAL`` error is returned.
```

Comment:
I think we missed adding `.. versionadded:: 1.46.0` to the new methods in the previous commit. We should add them here.

Reply:
Good point, I'll do that.

Are you okay with the flag's name? I don't love it but I couldn't think of anything better.

Reply:
> Are you okay with the flag's name? I don't love it but I couldn't think of anything better.

My only problem with the is that's Unix only but we have some Windows flags on other methods so... Do we know what error windows returns when the size of the pipe name is larger than 256? If it's something we translate as `UV_EINVAL` we might be just name it `UV_PIPE_UNIX_NO_TRUNCATE` and make no changes on the Win side.

Reply:
> Do we know what error windows returns when the size of the pipe name is larger than 256?

I'll try it on the CI and see what happens.

Reply:
Something that translates to UV_EACCES. ERROR_NOACCESS or ERROR_CANT_ACCESS_FILE, most likely.

I'm inclined to keep the current behavior unchanged and let users opt in through the flag. Agree/disagree?

Reply:
> I'm inclined to keep the current behavior unchanged and let users opt in through the flag. Agree/disagree?

Fair enough. :+1: